main/SetupMenu.cpp

```diff
@@ -947,7 +953,7 @@ void SetupMenu::audio_menu_create( MenuEntry *audio ){
 	audios->addCreator(audio_menu_create_tonestyles);
 
 	update_rentry(0);
-	audio_range_sm = new SetupMenuSelect( PROGMEM"Range", RST_NONE, 0 , true, &audio_range  );
+	audio_range_sm = new SetupMenuSelect( PROGMEM"Range", RST_NONE, audio_setup_s, true, &audio_range  );
```

Comment:
I remember changing this on the fly cause restarts, but lets see if this is still the case with the current state.